src/components/hand-poses.js

```diff
@@ -0,0 +1,125 @@
+const POSES = {
+  open: "allOpen",
+  thumbDown: "thumbDown",
+  indexDown: "indexDown",
+  mrpDown: "mrpDown",
+  thumbUp: "thumbsUp",
+  point: "point",
+  fist: "allGrip",
+  pinch: "pinch"
+};
+
+AFRAME.registerComponent("hand-poses", {
```

Comment:
Haven't fully worked through it but it seems like this component could be much cleaner if you handled each hand's pose in its's own instance of the component, since their states are completely independant anyway. It would remove a lot of the duplicate code and wierd dynamic property access going on.